main.py

```diff
@@ -6,12 +6,24 @@
 import sys
 import os
 import shutil
-import tarfile
-import urllib.request
-import io
 import argparse
 import code
 
+DEFAULT_CONFIG = {
+	"package-path": [
+		"packages",
+		os.path.join(os.path.abspath(os.path.dirname(__file__)), "packages")
+	],
+	"repos": ["https://halibot.fish:4842"],
+	"agent-instances": {},
+	"module-instances": {},
+	"cli-instances": {
+		"pm": {
+			"of": "core:PackageManager"
+		}
+	}
+}
```

Comment:
I think we should make this a file and add an option for `init` to specify the default config.